.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,69 @@
+{
+	"name": "vets-api native setup",
+	"image": "mcr.microsoft.com/devcontainers/base:bullseye",
+
+	"features": {
+		"ghcr.io/devcontainers-contrib/features/ruby-asdf:0": {
+			"version": "3.2.3"
+		},
+		"ghcr.io/robbert229/devcontainer-features/postgresql-client:1": {
+			"version": "15"
+		},
+		"ghcr.io/devcontainers-contrib/features/redis-homebrew:1": {
+			"version": "6.2"
+		}
+	},
+
+	"forwardPorts": [
+		3000,
+		9293,
+		5432,
+		6379
+	],
+	"portsAttributes": {
+		"3000": {
+			"label": "vets-api",
+			"onAutoForward": "notify",
+			"requireLocalPort": true
+		},
+		"9293": {
+			"label": "vets-api-healthcheck",
+			"onAutoForward": "notify",
+			"requireLocalPort": true
+		},
+		"5432": {
+			"label": "postgis",
+			"onAutoForward": "notify",
+			"requireLocalPort": true
+		},
+		"6379": {
+			"label": "redis",
+			"onAutoForward": "notify",
+			"requireLocalPort": true
+		}
+	},
+
+	"postCreateCommand": "sh .devcontainer/post-create.sh",
+	"postStartCommand": "sh .devcontainer/post-start.sh",
+
+	"customizations": {
+		"codespaces": {
+			"repositories": {
+				"department-of-veterans-affairs/vets-api-mockdata": {
+					"permissions": {
+						"contents": "read",
+						"pull_requests": "write"
+					}
+				}
+			}
+		},
+	    "vscode": {
+			"extensions": ["ms-azuretools.vscode-docker", "Shopify.ruby-lsp"]
+		}
+	},
+	"hostRequirements": {
+		"cpus": 8,
+		"memory": "32gb",
```

Comment:
Is this the minimum host requirement? This option prevents spinning up anything smaller which I assume would cost more.

Reply:
ah, I hadn't realized that it disallows selecting a smaller config. I'll remove this section.